terraform/cyhy_reporter_ec2.tf

```diff
@@ -105,3 +89,29 @@ resource "aws_volume_attachment" "cyhy_reporter_data_attachment" {
 
   skip_destroy = true
 }
+
+# Provision the reporter EC2 instance via Ansible
+module "cyhy_reporter_ansible_provisioner" {
+  source = "github.com/cloudposse/terraform-null-ansible"
+
+  # Ensure all EBS volumes are attached before running Ansible
```

Comment:
Same as above:
```suggestion
  # Ensure the EBS volume is attached before running Ansible
```

Reply:
Does [7a41648](https://github.com/cisagov/cyhy_amis/pull/422/commits/7a41648c06164c5973e68c7b7277f1974937a48e) also satisfy this?